incognia.go

```diff
@@ -7,11 +7,17 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"runtime"
 	"time"
+
+	"github.com/matishsiao/goInfo"
```

Comment:
I just took a look at this package and it doesn't comply with our standards.

Actually, this is so simple that we can write these functions ourselves. I don't believe we need a dependency for this.